networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc

```diff
@@ -6,14 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin.
+As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin using the _offline_ migration method or the _live_ migration method.
```

Comment:
As we already supported live migration, do we still need to keep the offline migration?

Reply:
We will support both. Live migration cannot cover all user's setups.